workers/unity/Packages/io.improbable.gdk.debug/WorkerInspector/Codegen/Tests/VisualElementConcealerTests.cs

```diff
@@ -0,0 +1,62 @@
+using NUnit.Framework;
+using UnityEngine.UIElements;
+
+namespace Improbable.Gdk.Debug.WorkerInspector.Codegen.EditmodeTests
+{
+    [TestFixture]
+    public class VisualElementConcealerTests
+    {
+        [TestCase(false)]
+        [TestCase(true)]
+        public void HideCollectionEventSetsHideProperty(bool value)
```

Comment:
I don't think this is mentioned in the style guide, but we tend to break naming convention on tests to make them more legible in the test runners, something like: 

```
HideCollectionEvent_sets_HideIfEmpty
```

Common formats are:

```
<thing>_<action>_when_<condition>
<thing>_when_<condition>_does_<action>
```